x-pack/plugins/uptime/server/lib/requests/get_certs.ts

```diff
@@ -5,23 +5,39 @@
  */
 
 import { UMElasticsearchQueryFn } from '../adapters';
-import { Cert, GetCertsParams } from '../../../../../legacy/plugins/uptime/common/runtime_types';
+import {
+  CertResult,
+  GetCertsParams,
+} from '../../../../../legacy/plugins/uptime/common/runtime_types';
 
-export const getCerts: UMElasticsearchQueryFn<GetCertsParams, Cert[]> = async ({
+export const getCerts: UMElasticsearchQueryFn<GetCertsParams, CertResult> = async ({
```

Comment:
In testing this it looks like we need to start filtering for data from newer heartbeat versions that contain the x509 fields. The top (blank) field in the screenshot below is what happens when this page tries to render data from heartbeat 7.7

![image](https://user-images.githubusercontent.com/131427/80387442-44300800-886e-11ea-93c8-664379f06c48.png)

Reply:
thanks, i will add exist filters.